packages/wonder-blocks-dropdown/components/separator-item.js

```diff
@@ -9,17 +9,31 @@ import Color from "@khanacademy/wonder-blocks-color";
 import Spacing from "@khanacademy/wonder-blocks-spacing";
 import {View} from "@khanacademy/wonder-blocks-core";
 
+import type {StyleType} from "@khanacademy/wonder-blocks-core";
+
 /**
  * A separator used in a dropdown menu.
  */
-export default class SeparatorItem extends React.Component<{}> {
+export default class SeparatorItem extends React.Component<{|
+    /**
+     * In case we use react-window, this needs to be added in order to inject
+     * styles to calculate the position
+     * @ignore
+     */
+    style?: StyleType,
+|}> {
     static __IS_SEPARATOR_ITEM__ = true;
     static isClassOf(instance: React.Element<any>) {
         return instance && instance.type && instance.type.__IS_SEPARATOR_ITEM__;
     }
 
     render() {
-        return <View style={styles.separator} />;
+        return (
+            //  wrapper to use styles from react-window (if applies)
+            <View style={this.props.style}>
```

Comment:
Do we want this to be a wrapper or is it enough to do: `<View style={[this.props.style, styles.separator]} />`

Reply:
We can merge both styles if we use `box-shadow` instead of `background`. I'll make that change.